src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -556,6 +559,24 @@ def _get_hash_machine_id():
     return str(uuid.UUID(_get_hash_mac_address()[0:32]))
 
 
+@decorators.suppress_all_exceptions(fallback_return='')
+def _get_distro_name():
+    try:
+        import distro
+        return distro.name(pretty=True)
+    except ImportError:
+        return ''
```

Comment:
Do we really need this try except clause given all expressions are already suppressed by `decorators.suppress_all_exceptions`?

Reply:
Without this try except, IDE will remind error
![image](https://user-images.githubusercontent.com/10227683/215731709-70dc3b3e-ee98-4cdc-b8e5-f00b348fad55.png)

But with try except, it only reminds warning
![image](https://user-images.githubusercontent.com/10227683/215731407-1f2acc54-b710-4b04-985c-04f500c886f2.png)

I don't like the error reminder so...😂